src/Connection/Http1Connection.php

```diff
@@ -35,6 +36,7 @@
 use Amp\TimeoutCancellationToken;
 use function Amp\asyncCall;
 use function Amp\call;
+use function Amp\Internal\getCurrentTime;
```

Comment:
@trowski We should move `getCurrentTime()` to `Amp\getCurrentTime()` before merging this.

Reply:
Depends on https://github.com/amphp/amp/pull/290.